File: articles/algorithms-in-public-policy-and-governance/go/procedural_readiness.go
package main

import "fmt"

func main() {
	dueProcess := 0.58
	transparency := 0.52
	humanReview := 0.60
	appealReadiness := 0.54
	score := (dueProcess + transparency + humanReview + appealReadiness) / 4.0
	fmt.Printf("procedural_readiness_score=%.4f\n", score)
}

File: articles/algorithms-in-public-policy-and-governance/rust/procedural_readiness.rs
fn main() {
    let due_process = 0.58_f64;
    let transparency = 0.52_f64;
    let human_review = 0.60_f64;
    let appeal_readiness = 0.54_f64;
    let score = (due_process + transparency + human_review + appeal_readiness) / 4.0;
    println!("procedural_readiness_score={:.4}", score);
}
